src/models/storage/batch.py

```diff
@@ -18,31 +18,36 @@
 from pandas import DataFrame
 
 from src.models.inference.outcome import Outcome
+from src.utils.logging_manager import LoggingManager, LoggingLevel
 
 
 class Batch:
     """
     Data model used for storing a batch of frames
 
     Arguments:
-        frames (DataFrame): List of video frames
-        info (FrameInfo): Information about the frames in the batch
+        frames (List[Dict], DataFrame): List of frames
         outcomes (Dict[str, List[BasePrediction]]): outcomes of running a udf
         with name 'x' as key
         identifier_column (str): A column used to uniquely a row
 
 
     """
 
-    def __init__(self, frames: DataFrame, outcomes=None, temp_outcomes=None,
+    def __init__(self, frames, outcomes=None, temp_outcomes=None,
```

Comment:
keep the frames type as DataFrame

Reply:
Also check whether line 44-50 is necessary.

Reply:
Fixed.
Those lines are enforcing that frames are indeed dataframe.  So keeping it